test/cpp/jit/test_flatbuffer.cpp

```diff
@@ -713,36 +713,42 @@ TEST(FlatbufferTest, DefaultArgsConv) {
 
   std::vector<torch::jit::IValue> inputs;
 
-  Module m("m");
-  m.register_parameter("weight", torch::ones({20, 1, 5, 5}), false);
-  m.register_parameter("bias", torch::ones({20}), false);
-  m.define(R"(
-    def forward(self, input):
-      return torch.conv2d(input, self.weight, self.bias, [1, 1], [0, 0], [1, 1], 1)
-  )");
+//  Module m("m");
```

Comment:
I will reset this file. Wanted to check mobile net model is serialized write way.